src/supported-value.js

```diff
@@ -76,5 +157,11 @@ export default function supportedValue(property, value) {
   // Reset style value.
   el.style[property] = ''
 
+  if (isArray) {
+    return stringToArray(cache[cacheKey])
```

Comment:
can we use already something we have already instead of converting?
same for the next line